models/event_filters/stripe_charges.sql

```diff
@@ -5,7 +5,7 @@ with charges as (
 )
 
 select
-  id,
+  data__object__id as id,
```

Comment:
(can't attach this to line 16, since it didn't change, but:)

With your WHERE statement, there can be multiple events for each charge (due to refunds, disputes, or metadata updates)... to make this a unique list of charges, replace line 16 with:

``` SQL
where "type" in ('charge.succeeded', 'charge.failed')
```

I had been thinking about this table as an immutable "charge events" table, which is different from a list of charge objects, which have states and need to be updated. It looks like stitch provides the stripe_charges table in the integration, which might be a preferable way to build this. The Dispute and Refund objects are missing from stitch, but I think you can rebuild the important pieces by treating `stripe_balance_history` as a second events stream